cert.go

```diff
@@ -144,7 +144,25 @@ func loadCAFromConfig(l *logrus.Logger, c *config.C) (*cert.NebulaCAPool, error)
 		}
 	}
 
+	var expired int
 	CAs, err := cert.NewCAPoolFromBytes(rawCA)
+	if errors.Is(err, cert.ErrExpired) {
+		for _, cert := range CAs.CAs {
+			if cert.Expired(time.Now()) {
+				expired++
+				fingerprint, _ := cert.Sha256Sum()
+				l.WithField("name", cert.Details.Name).
```

Comment:
We currently have two patterns for logging certificates.

What you have here closely models handshake log messages, although we'd want to correct for `camelCase` and naming conventions in place today. https://github.com/slackhq/nebula/blob/949ec7865335bfdf1c8977a22787f96873713205/handshake_ix.go#L109-L113

So if we want to follow that pattern, then:

```go
l.WithField("certName", cert.Details.Name).
        WithField("notBefore", cert.Details.NotBefore).
        WithField("notAfter", cert.Details.NotAfter).
        WithField("fingerprint", fingerprint).
        WithField("signature", fmt.Sprintf("%x", cert.Signature)).
        Warn("expired certificate present in CA pool")
```

However, there are a few other spots that log the full cert, which includes all the above information. If you are running a text mode logger it doesn't look super great though. logrus might have a way to make this better though (not a now problem)

https://github.com/slackhq/nebula/blob/949ec7865335bfdf1c8977a22787f96873713205/interface.go#L267

I like how succinct it is though:

```go
l.WithField("cert", cert).Warn("expired certificate present in CA pool")
```